src/ngx_http_lua_kong_log.c

```diff
@@ -87,5 +92,11 @@ ngx_http_lua_kong_get_dynamic_log_level(ngx_uint_t current_log_level)
 int
 ngx_http_lua_kong_ffi_get_dynamic_log_level(int current_log_level)
 {
-    return ngx_http_lua_kong_get_dynamic_log_level(current_log_level);
+    int log_level = ngx_http_lua_kong_get_dynamic_log_level(current_log_level);
+
+    if (log_level == NGX_LOG_DEBUG_ALL) {
+        log_level = NGX_LOG_DEBUG;
+    }
```

Comment:
for the api upper user, we need to translate it from 0x7fffff... to log level number like 0~8.